examples/chainermn/mnist/train_mnist.py

```diff
@@ -106,7 +114,10 @@ def main():
     # Some display and output extensions are necessary only for one worker.
     # (Otherwise, there would just be repeated outputs.)
     if comm.rank == 0:
-        trainer.extend(extensions.DumpGraph('main/loss'))
+        if device.xp is not chainerx:
+            # Disabled for ChainerX.
+            # See examples/mnist/train_mnist.py
```

Comment:
Maybe we can figure out why it is temporally disabled in chainer example https://github.com/chainer/chainer/blob/edc929818b0021543d2b8dc773aa89b41ec06d09/examples/mnist/train_mnist.py#L97

Reply:
I'm now consulting @niboshi -san.

Reply:
`DumpGraph` is disabled because ChainerX doesn't have a public API set to traverse computational graphs.

Reply:
OK, understood! Thanks!

Reply:
Can you please add a comment about this?

Reply:
fixed!